CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [v0.4.0]
+
+### Added
+- Added new CoreDNS version 1.4.0 ([release notes](https://coredns.io/2019/03/03/coredns-1.4.0-release/)).
```

Comment:
During the VOO spec we agreed to have changelog entries in imperative form. This would be:

```suggestion
- Add new CoreDNS version 1.4.0 ([release notes](https://coredns.io/2019/03/03/coredns-1.4.0-release/)).
```

etc.

Thanks for starting changelog here. This is cool.

Reply:
We're going a bit off topic but...

No strong opinions on whether we use imperative or past tense. But I do care a lot about consistency. If we're using imperative we should use it everywhere and update `fmt`.

https://github.com/giantswarm/fmt/blob/master/releases/how_write_changelog_message.md